lib/util.py

```diff
@@ -402,6 +402,22 @@ def safe_boolcomp(value, expected):
     return str(value).lower() == str(expected).lower()
 
 
+# DEV NOTE: merging of creation.lib.CWParamDict.is_true() and factory.tools.OSG_autoconf.is_true()
+#   the first one required the argument to be a string. OK to drop that
+def is_true(value):
+    """Case-insensitive "True" string parsing helper.
+    Return True for true (case-insensitive string representation matching), False otherwise.
+
+    Args:
+        value: argument to evaluate as True or False. Can be any type.
```

Comment:
What does `value: .... Can be of any type.` mean? If `value` can be of any type, using `str(value)` may not always be valid, right?